examples/spark/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+import java.util.Properties
+
+plugins {
+    kotlin("jvm") version "1.3.70"
```

Comment:
This plugin should be included in all sub-projects already. See the `examples/spring` build file

https://github.com/ExpediaGroup/graphql-kotlin/blob/2806cd80125dbba451150e42bbf700589649f923/examples/spring/build.gradle.kts#L3